contracts/payoutStrategy/MerklePayoutStrategy/MerklePayoutStrategyFactory.sol

```diff
@@ -54,7 +54,9 @@ contract MerklePayoutStrategyFactory is OwnableUpgradeable {
   function create(
   ) external returns (address) {
 
-    address clone = ClonesUpgradeable.clone(payoutImplementation);
+    bytes32 salt = keccak256(abi.encodePacked(msg.sender));
```

Comment:
Does this actually work more than once? 

>  @dev Deploys and returns the address of a clone that mimics the behaviour of `implementation`.
> This function uses the create2 opcode and a `salt` to deterministically deploy
>  the clone. Using the same `implementation` and `salt` multiple time will revert, since
>  the clones cannot be deployed twice at the same address.

i think we should add an internal nonce to create salt using the nonce and the sender